src/main/webapp/js/message-loader.js

```diff
@@ -11,6 +11,17 @@ function fetchUserEmail() {
   $.ajaxSetup({ async: true });
 }
 
+async function getUserProfileUrl(email) {
```

Comment:
No need to put the async keyword here since the function is returning a Promise. If this keyword is set, the code inside will run synchronously (refer to: https://stackoverflow.com/a/45594870/7011580)

Reply:
Ok sure!